marlowe-dashboard-client/src/Icons.purs

```diff
@@ -13,15 +14,23 @@ module Material.Icons
   , previous
   , roles
   , terms
+  , timer
+  , timer'
   , wallet
   ) where
 
 import Prelude
 import Halogen.HTML (ClassName(ClassName), HTML, span, text)
-import Halogen.HTML.Properties (class_)
+import Halogen.HTML.Properties (classes)
+import Data.Array (cons)
 
 icon :: forall p i. String -> HTML p i
-icon str = span [ class_ $ ClassName "material-icons-round" ] [ text str ]
+icon str = icon' str []
+
+-- The apostrophe helpers should we used when we need to apply other styles to the icons, as wrapping them
+-- between a div or another span generates weird vertical alignment issues.
```

Comment:
Should we do `icon` and `icon_` (analogous to Halogen's `div`/`div_`)?

Reply:
Sure, we can do that change in a separate PR, but keep in mind that if we want to keep consistency, it will change current icons, as the version that doesn't receive extra options should be the `icon_` one.